monai_ex/transforms/utility/dictionary.py

```diff
@@ -423,6 +424,28 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
         return d
 
 
+class ExtractCenterlined(Randomizable, MapTransform):
+    """Dictionary-based version :py:class:`monai_ex.transforms.ExtractCenterline`."""
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        label_key: str,
+        allow_missing_keys: bool = False
```

Comment:
`keys` is not used? label_key is not necessary. Just use `keys` instead of specifiying a `label_key`.